tools/topology/sof-hda-generic-idisp.m4

```diff
@@ -0,0 +1,97 @@
+#
+# Topology for generic Apollolake UP^2 with pcm512x codec and HDMI.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
```

Comment:
and I thinking this too but not sure.

Reply:
not sure how, this is the same in the hda-generic part. and while it's a specific case it's not really broken in the sense that the same settings are used for HDMI in all platforms anyways.